projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java

```diff
@@ -358,56 +309,54 @@ public static void saveTrips(Path csvPath, String path) {
           isHeader = false;
           continue;
         }
+        if(!routesId.contains(Integer.parseInt(record.get("route_id"))) || !servicesId.contains(Integer.parseInt(record.get("service_id")))){
+          continue;
+        }
         Trip.Builder trip = Trip.newBuilder();
-        trip.setRouteId(Integer.parseInt(record.get("route_id")))
+        trips.add(trip.setRouteId(Integer.parseInt(record.get("route_id")))
             .setServiceId(Integer.parseInt(record.get("service_id")))
             .setTripId(record.get("trip_id"))
             .setTripHeadsign(record.get("trip_headsign"))
             .setDirectionId(Integer.parseInt(record.get("direction_id")))
             .setShapeId(record.get("shape_id"))
-            .build()
-            .writeDelimitedTo(output);
+            .build());
       }
+      return trips;
     } catch (IOException e) {
       throw new RuntimeException(e);
     }
   }
 
+      // Paths to the CSVs files
+      static Path calendarTxt;
```

Comment:
Please follow this guide on file naming:
https://github.com/hasadna/hasadna/blob/master/NOMENCLATURE.md
I think it should be `Txt->absTxtFilename` or `Txt->absFilename`.